PovioKit/Classes/Utilities/PromiseKit/Promise.swift

```diff
@@ -114,3 +225,20 @@ public extension Promise {
     }
   }
 }
+
+extension Promise where Value == Void {
+  func resolve() {
+    resolve(with: ())
+  }
+}
+
+// MARK: - Private methods
+private extension Promise {
+  static func areAllSuccessful(_ promises: [Promise<Value, Error>]) -> Bool {
```

Comment:
The term you are looking for is `fulfilled`:
`areAllSuccessful` -> `areAllFulfilled`